osc-staging.py

```diff
@@ -336,7 +337,7 @@ def do_staging(self, subcmd, opts, *args):
             [--merge] [--try-strategies] [--strategy]
             [STAGING...] [REQUEST...]
         osc staging unselect [--cleanup] [-m MESSAGE] [REQUEST...]
-        osc staging unlock
+        osc staging [-p PROJECT] unlock
```

Comment:
Including a global option on one subcommand and does not make sense. The global options are documented first and the subcommand specific ones last. Having this documentation split up by subcommand might be nice, but the current implementation does not lend it self to that.

If anything perhaps some note about the `--project` option at the beginning of the documentation. Could also mention the feature I added to allow detecting project based on `.osc/project` from current directory (which is how I work along with PS1 to indicate as much).

Reply:
It may also make sense not to include the `osc staging` bit in front of the subcommands to make it more clear this does not show the global options.

Reply:
Makes sense, i was a bit to fast with the PR. The reason for using the unlock command was that i wanted to select a package to one of the adi's. By accident i just pasted a few spaces and entered. After cancelling with control-c there was an lock acquired by me. After like 10-15 minutes i unlocked it manually.

Reply:
Another behavior possibly worth documenting is that pressing `ctrl-c` once will kill and allow unlocking, but pressing more than once will immediately kill and leave the lock.

Reply:
To clarify how I work https://asciinema.org/a/d8jEi7ZzxjrvQbs88clK0WOB0 which may be worth documenting.